processing/src/main/java/org/apache/carbondata/processing/loading/DataLoadProcessBuilder.java

```diff
@@ -76,6 +78,19 @@ public AbstractDataLoadProcessorStep build(CarbonLoadModel loadModel, String[] s
     }
   }
 
+  private AbstractDataLoadProcessorStep buildInternalForAvroSDKLoad(CarbonIterator[] inputIterators,
```

Comment:
THis logic is same as `buildInternalForPartitionLoad` then why special method and handling required? why not use same partition flow?

Reply:
Both of Partition and AVRO calling the same method now.